src/util/records.py

```diff
@@ -1245,10 +1362,20 @@ def replace_record_references_batch(cr, id_mapping, model_src, model_dst=None, r
 
 def replace_in_all_jsonb_values(cr, table, column, old, new, extra_filter=None):
     """
-    Will replace `old` by `new` (whole word match) in all jsonb value of `column` of `table`.
-    The `extra_filter` should use the `t` alias. It can also include `{parallel_filter}` to
-    execute the query in parallel.
-    `old` can be a simple term (str) or a regexp (util.PGRegexp)
+    Replace values in JSONB columns.
+
+    This function replaces `old` by `new` in JSONB values. It is useful for replacing
+    values in *all* translations of translated fields.
+
+    :param str table: table name where the values are to be replaced
+    :param str column: column name where the values are to be replaced
+    :param str old: original value to replace, can be a simple term (str) or a Postgres
+                    regular expression wrapped by :class:`~odoo.upgrade.util.pg.PGRegexp`
+    :param str new: new value to set, can be a simple term (str) or a Postgres regular
+                    expression wrapped by :class:`~odoo.upgrade.util.pg.PGRegexp`
```

Comment:
A regex for the `new` value doesn't make sense.
```suggestion
    :param str new: new value to set
```